src/index.js

```diff
@@ -25,7 +26,7 @@ module.exports = async function({
   }
   await page.setContent(html)
   const element = await page.$('body')
-  const buffer = await element.screenshot({ path: output, type, omitBackground: transparent, encoding })
+  const buffer = await element.screenshot({ path: output, type, quality: quality, omitBackground: transparent, encoding })
```

Comment:
if the variable name is the same you don't need to repeat it 👍 

```suggestion
  const buffer = await element.screenshot({ path: output, type, quality, omitBackground: transparent, encoding })
```

Reply:
@frinyvonnick thanks for the feedback. Fixed the repetition.